src/main/java/self/project/messaging/security/AccountFullDto.java

```diff
@@ -0,0 +1,24 @@
+package self.project.messaging.security;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.List;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+public class AccountFullDto {
+    private Long id;
+    private String password;
+    private String phoneNumber;
+    private String username;
+    private String role;
+
+    private List<Long> chats;
```

Comment:
Since it's only used in security stuff, chats can be deleted. And this class can probably be turned into a record as well.